documentation/docs/student/calendar.mdx

```diff
@@ -0,0 +1,86 @@
+title: Calendar
+sidebar_label: Calendar
+---
+
+import calendarTabImage from './assets/calendar/calendar-tab.png';
+import desktopMonthPresentationImage from './assets/calendar/desktop-month-presentation.png';
+import desktopWeekPresentationImage from './assets/calendar/desktop-week-presentation.png';
+import mobileMonthPresentationImage from './assets/calendar/mobile-month-presentation.png';
+import mobileDayPresentationImage from './assets/calendar/mobile-day-presentation.png';
+import eventDetailsImage from './assets/calendar/event-details.png';
+import desktopSubscriptionComponent from './assets/calendar/desktop-subscription-component.png';
+import mobileSubscriptionComponent from './assets/calendar/mobile-subscription-component.png';
+import desktopFilterComponent from './assets/calendar/desktop-filter-component.png';
+import mobileFilterComponent from './assets/calendar/mobile-filter-component.png';
+import desktopNavigationControls from './assets/calendar/desktop-navigation-controls.png';
+import mobileNavigationControls from './assets/calendar/mobile-navigation-controls.png';
+import Image, {ImageSize} from "../../src/components/Image/Image";
+
+# Calendar
+Artemis features a calendar that allows users to quickly view all important events related to a specific course.
+
+## General Information
+The Artemis calendar can be found in the Calendar tab of the course overview.
+It displays important events related to lectures, tutorials, exercises, and exams.
+
+<Image src={calendarTabImage} alt={"Calendar Tab in Course Overview Sidebar"} size={ImageSize.small}></Image>
+
+The following table shows the dates that are displayed as calendar events, if available.
+Note that an event can either represent a specific point in time (in which case it only has a start date) or a time period (in which case it has both a start and an end date).
+
+| Entity  | Dates                                                            |
+|----------|------------------------------------------------------------------|
+| Lecture  | start, end  |
+| Tutorial  | start, end  |
+| Exercise  | release, start, due, assessment due  |
+| Exam  | start, end, results release, student review start, student review end |
+
+## Switching between Presentations
+To view events in the calendar, users can switch between a high-level overview and a more detailed presentation.
+The available views differ between small and large devices to optimize the calendar layout for each screen size.
+
+On larger devices, such as laptops and desktop computers, the calendar offers a high-level **month** presentation and a more detailed **week** presentation.
+
+<Image src={desktopMonthPresentationImage} alt={"Month Presentation for Large Devices"}  caption={"Month Presentation for Large Devices"} size={ImageSize.large}></Image>
+<Image src={desktopWeekPresentationImage} alt={"Week Presentation for Large Devices"} caption={"Week Presentation for Large Devices"} size={ImageSize.large}></Image>
```

Comment:
Can we hide the border for these pictures? I do not think they need them as there already is a border within the screenshot